package.json

```diff
@@ -71,12 +72,12 @@
     "prettier": "^2.0.5",
     "react-scripts": "^3.3.0",
     "react-test-renderer": "16.13.1",
-    "styled-components": "^4.3.2",
+    "styled-components": "^4.4.1",
```

Comment:
I looked at the change log and it seems the changes on snapshots are expected, also this is an important release before v5 which is blocking some other updates like jest